src/libs/Share/index.android.js

```diff
@@ -2,8 +2,12 @@ import ShareMenu from 'react-native-share-menu';
 import Navigation from '../Navigation/Navigation';
 import isShareExtension from './isShareExtension';
 import navigateToShare from './navigateToShare';
+import ROUTES from '../../ROUTES';
 
-const dismiss = () => Navigation.dismissModal();
+const dismiss = (reportID) => {
+    Navigation.dismissModal();
```

Comment:
This looks really smooth already, but from looking at the `dismissModal` source, I think we can pass the reportID as an argument to `dismissModal` instead of needing to make a separate navigation call. (I found this because I was looking to see if there was a way we can combine the navigation actions, since multiple separate steps might look janky sometimes)

Reply:
Ah! I missed that it takes parameters (TS come swiftly). That's an easy update haha